python/cudf_polars/tests/test_join.py

```diff
@@ -16,10 +16,6 @@
         "left",
         "semi",
         "anti",
-        pytest.param(
-            "cross",
-            marks=pytest.mark.xfail(reason="cross join not implemented"),
-        ),
```

Comment:
Note there's no need to parametrize over all of these things for the cross join case, because they are all ignored.